content/en/docs/instrumentation/js/exporters.md

```diff
@@ -18,25 +18,32 @@ Jaeger) you'll want to use an exporter package, such as
 `@opentelemetry/exporter-trace-otlp-http`:
 
 ```shell
-npm install --save @opentelemetry/exporter-trace-otlp-http
+npm install --save @opentelemetry/exporter-trace-otlp-http \
+  opentelemetry/exporter-metrics-otlp-grpc
 ```
 
 Next, configure the exporter to point at an OTLP endpoint. For example you can
-update `tracing.ts|js` from the
+update `instrumentation.ts|js` from the
 [Getting Started](/docs/instrumentation/js/getting-started/nodejs/) like the
 following:
 
 <!-- prettier-ignore-start -->
 {{< tabpane langEqualsHeader=true >}}
 {{< tab Typescript >}}
 /*tracing.ts*/
-import * as opentelemetry from '@opentelemetry/sdk-node';
+import * as opentelemetry from "@opentelemetry/sdk-node";
 import {
   getNodeAutoInstrumentations,
 } from "@opentelemetry/auto-instrumentations-node";
 import {
   OTLPTraceExporter,
 } from "@opentelemetry/exporter-trace-otlp-http";
+import {
+  OTLPMetricExporter
+} from "@opentelemetry/exporter-metrics-otlp-http";
```

Comment:
Above we recommend installing the gRPC exporter, but here we're using oltp/json. I wonder if this example should be using the `@opentelemetry/exporter-metrics-otlp-proto` and `@opentelemetry/exporter-trace-otlp-proto` packages instead as otlp/json is not really widely accepted.

Reply:
Oh, that grpc/http thing was a mix up, I wanted to use http for both, because I wanted to align it with what is done for the OTLPTraceExporter.

I can change both to proto then?

Reply:
done